rules/endpoint/elastic_endpoint.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint alert is received. Enabling this rule allows you to
+immediately begin investigating your Elastic Endpoint alerts.
+"""
+enabled = true
+from = "now-15m"
+index = ["logs-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Elastic Endpoint"
+risk_score = 47
+rule_id = "9a1a2dae-0b5f-4c3d-8305-a268d404c306"
+rule_name_override = "message"
+severity = "medium"
+tags = ["Elastic", "Endpoint"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.kind:alert and event.module:(endpoint and not endgame)
+'''
+
+
+[[rule.risk_score_mapping]]
+field = "event.risk_score"
+operator = "equals"
+value = ""
+
+[[rule.severity_mapping]]
+field = "event.severity"
+operator = "equals"
+value = "21"
+severity = "low"
+
+[[rule.severity_mapping]]
+field = "event.severity"
+operator = "equals"
+value = "47"
+severity = "medium"
+
+[[rule.severity_mapping]]
+field = "event.severity"
+operator = "equals"
+value = "73"
+severity = "high"
+
+[[rule.severity_mapping]]
+field = "event.severity"
+operator = "equals"
+value = "99"
+severity = "critical"
```

Comment:
Need to confirm the `SeverityMapping` for the `Elastic Endpoint` rule. `signal.rule.severity` is a `keyword` and we've been using `low`/`medium`/`high`/`critical` as an enum for it, however `event.severity` (and `signal.original_event.severity`) are mapped as a `long`.

Reply:
These are fine for now -- from product:

> Malware on-write and on-execution when Detected, is a critical/99
Malware on-write and on-execution when Prevented, is a high/73